exercises/game-of-life/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+[Conway's Game of Life][game-of-life] is a fascinating cellular automaton created by the British mathematician John Horton Conway in 1970. The game consists of a two-dimensional grid of cells that can either be "alive" or "dead." Each generation, the cells interact with their eight neighbors, which are cells adjacent horizontally, vertically, or diagonally.
```

Comment:
Could you use one line per sentence?

```suggestion
[Conway's Game of Life][game-of-life] is a fascinating cellular automaton created by the British mathematician John Horton Conway in 1970.
The game consists of a two-dimensional grid of cells that can either be "alive" or "dead."
Each generation, the cells interact with their eight neighbors, which are cells adjacent horizontally, vertically, or diagonally.
```

Reply:
Maybe you briefly explain what a cellular automaton is?

Reply:
I'd also suggest to copy the last line to the `instructions.md` file and update the text here to something like:

```
Each generation, the cells interact with their eight neighbors via a set of rules, which defined the new generation.
```

This is a bit rough, but you probably get what I'm trying to do here :)

Reply:
Something like this?